packages/components/src/components/search/_search.scss

```diff
@@ -89,20 +89,20 @@
   }
 
   // Large styles
-  .#{$prefix}--search--lg .#{$prefix}--search-input,
-  .#{$prefix}--search--lg.#{$prefix}--search--expandable.#{$prefix}--search--expanded
+  .#{$prefix}--search--md .#{$prefix}--search-input,
```

Comment:
Do we need to worry about any of these changes for component implementations like Vue/Angular? 👀

Reply:
Good point, should probably give them a heads up. I believe the `Button` update also changes 2 class names (`xl` --> `2xl`, `lg` --> `xl`)

Reply:
Actually, on second thought, to avoid problems downstream, I just reverted it back to the original classes and added notes to update the class down the road. I'll update the Button PR in the same way